src/docs/about.md

```diff
@@ -0,0 +1,186 @@
+system. This plugin systems allow behavior between input and output directories to be described. Broccoli then
+ensures plugins are called at the appropriate time, to ensure the build is successful, and writes the result to
+a `target` directory. Each plugin is responsible for processing files passed to it in input directories, and 
+writing files to its output directory.
+
+Broccoli doesn't really care about files, it simply takes source directories and passes them as inputs to 
+plugins, creates an output directory for the plugin to write to, and passes that output directory it as an input
+to the next plugin.
+
+Broccoli is configured with a file in the root of your project called `Brocfile.js`. This file defines the build
+pipeline for your application, and is written in plan old JavaScript. The order in which operations happen is
+determined by this build file.
+
+You can think of broccoli-plugins much like a simple programming language, where the output of a function can be 
+passed as the input(s) to another function.
+
+E.g.:
+
+```js
+let js = babel('src');
+js = uglify(js);
+
+let css = sass('src/styles', 'site.scss');
+css = autoprefixer(css);
+
+const result = merge([js, css]);
+
+module.exports = result;
+```
+This could also be expressed as:
+```js
+module.exports = merge([
+    uglify(
+        babel('src')
+    ),
+    autoprefixer(
+        sass('src/styles', 'site.scss')
+    )
+]);
+```
+
+In the above, a `src` directory is passed to the `babel()` plugin (which will convert our new ES6 syntax into 
+ES5 that runs in the browser), the output of that is passed to the `uglify()` plugin which will minify our JS 
+into a smaller format. The output of `uglify()` will in turn be passed as 1 input to `merge()`.
+
+Additionally, a `src/styles` directory and the input file `site.scss` is passed to `sass()` which will convert your
+`.scss` files into `.css` files, its output (your css) is passed as an input to `autoprefixer()` which will add
+vendor  prefixes (like `-ms` or `-webkit`) to attributes, which is then in turn passed into the `merge()` plugin.
+
+The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our 
+uglified JavaScript and out vendor prefixed css. This then becomes our final output and is what is written to 
+our target (destination) directory.
+
+This should all be fairly familiar to you if you've ever written JavaScript (or any programming language for that
+matter) before, it's just inputs and output.
+
+## Thinking in Broccoli
+
+There are 2 main concepts to get your head around when using Broccoli:
+
+* [Plugins](#plugins)
+* [Trees](#trees)
+
+### Plugins
+
+Plugins are what a build pipeline developer will interact with most. Plugins are what do the actual work of
+transforming input files into output files. The API of a plugin is pretty simple, all that's required is creating
```

Comment:
you may want to be careful of input/output as terminology, since they could be taken to mean pipeline imports or plugin imports.

```suggestion
transforming files at each step of build process. The API of a plugin is pretty simple, all that's required is creating
```